src/utils.js

```diff
@@ -349,9 +349,30 @@ const generateDetailTable = (metrics, rowData, {
 	return table;
 };
 
+const generateGui = (table) => {
+	const output = {
+		metrics: [],
+		repos: [],
+		rows: [],
+	};
```

Comment:
seems like this could do `table.map`, twice, to generate rows and repos, and then "metrics" could be done at creation time? (unless `table.options.head` has forEach and not map, and is also not arraylike so you couldn't pass it into Array.from)